README.md

```diff
@@ -168,6 +168,9 @@ overwritten by the VIRTUALENV_SYSTEM_SITE_PACKAGES environment variable. This fl
 We use `uv pip` to install packages into the virtual environment. The behavior of this can be configured via the
 following options:
 
+tox-uv resolves your project location based on the `package_root` setting. If that option is not defined, it will fall
```

Comment:
We don't really want to publicize `setupdir`, rather `package_root` is what we should only advertise.